charts/spire/charts/tornjak-frontend/templates/tests/test-tornjak-connection.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "tornjak-frontend.fullname" . }}-test-connection"
+  namespace: {{ include "tornjak-frontend.namespace" . }}
+  labels:
+  annotations:
+    "helm.sh/hook": test
+spec:
+  securityContext:
+    {{- toYaml .Values.podSecurityContext | nindent 4 }}
+  containers:
+    - name: wget-tornjak-frontend
+      image: busybox
+      command: ['wget']
+      args: ['--no-check-certificate', '-O', '/dev/null', 'http://{{ include "tornjak-frontend.service" . }}:{{ .Values.service.port }}']
```

```suggestion
    - name: netcat-tornjak-frontend
      image: cgr.dev/chainguard/bash:latest
      command: ['nc']
      args: ['-zvw3', '{{ include "tornjak-frontend.fullname" . }}', '{{ .Values.service.port }}']
```

or 

```suggestion
    - name: curl-tornjak-frontend
      image: cgr.dev/chainguard/bash:latest
      command: ['curl']
      args: ['-k', '-s', '-f', 'http://{{ include "tornjak-frontend.fullname" . }}.{{ include "tornjak-frontend.namespace" . }}.svc.cluster.local:{{ .Values.service.port }}']
```

To align with #175 which is ready for merge.